server/views/partials/news.hjs

```diff
@@ -1,6 +1,6 @@
 {{#news}}
     <article>
-        <h1 class="title">{{title}}</h1>
+        <h1 class="title"><a  href="post/{{url}}">{{title}}</a></h1>
```

Comment:
If you use an absolute path (```/post/{{url}}```) instead of a relative path, it will fix the 404 error.

Reply:
ofcourse! i completely forgot that. 0_0